Presentations/06-R-programming-practices.Rmd

```diff
@@ -714,25 +626,26 @@ knitr::include_graphics("img/statalog.png")
 ---
 
 # Functions inception  
-  
+
+## Piping
+
 * This is a __very powerful technique__, as you will soon see
 
 * It's __also a common source of error__, as you can only use one function inside the other if the output of the inner function is the same as the input of the outer function
 
 * It can also get quite tricky to follow what a line of code with multiple functions inceptions is doing
 
-* Which is why we sometimes use pipes: `%>%`
+* And that is why we use pipes: `%>%`
 
 ---
 
-# Functions inception  
+# Functions inception
+
+## Piping
```

Comment:
Do we need piping again?

Reply:
I think so, just to make sure that everyone understands well what they do. Will make sure to touch on this point very fast so we don't lose focus on the new contents